<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dice</class>
 <widget class="QWidget" name="dice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>389</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>238</width>
    <height>389</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>238</width>
    <height>389</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="bt_clear">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>96</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_exit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>96</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>231</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="sp_d2"/>
      </item>
      <item>
       <widget class="QPushButton" name="bt_d2">
        <property name="text">
         <string>d2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vlb_d2">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSpinBox" name="sp_d4"/>
      </item>
      <item>
       <widget class="QPushButton" name="bt_d4">
        <property name="text">
         <string>d4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vlb_d4">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSpinBox" name="sp_d6"/>
      </item>
      <item>
       <widget class="QPushButton" name="bt_d6">
        <property name="text">
         <string>d6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vlb_d6">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QSpinBox" name="sp_d10"/>
      </item>
      <item>
       <widget class="QPushButton" name="bt_d10">
        <property name="text">
         <string>d10</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vlb_d10">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QSpinBox" name="sp_d12"/>
      </item>
      <item>
       <widget class="QPushButton" name="bt_d12">
        <property name="text">
         <string>d12</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vlb_d12">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QSpinBox" name="sp_d20"/>
      </item>
      <item>
       <widget class="QPushButton" name="bt_d20">
        <property name="text">
         <string>d20</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vlb_d20">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QSpinBox" name="sp_d100"/>
      </item>
      <item>
       <widget class="QPushButton" name="bt_d100">
        <property name="text">
         <string>d100</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vlb_d100">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bt_exit</sender>
   <signal>clicked()</signal>
   <receiver>dice</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
